src/core/jwt/jwtTokenStore.js

```diff
@@ -55,6 +59,11 @@ const jwtTokenStoreFactory = function jwtTokenStoreFactory({
          * @returns {Promise<Boolean>} token successfully set
          */
         setAccessToken(token) {
+            if (usePerTokenTTL) {
+                const tokenPayload = parseJwtPayload(token);
+                const buffer = 10000;
```

Comment:
question:  Is it to ensure we ask to renew the token 10s before it expires?

Reply:
Yes that is the purpose. Since it's such a rare case (might realistically occur if a request is made in the final 2-3 seconds out of the 500 - and then the fallback mechanism works), I am tempted to get rid of this buffer and have simpler code.

Reply:
I took out the buffer now (inspired by Saint-Exupéry quote about less code is better).